packages/manager/apps/dedicated/client/app/app.js

```diff
@@ -43,6 +43,7 @@ import ovhManagerVps from '@ovh-ux/manager-vps';
 import ovhManagerVrack from '@ovh-ux/manager-vrack';
 import ovhManagerIplb from '@ovh-ux/manager-iplb';
 import { detach as detachPreloader } from '@ovh-ux/manager-preloader';
+import ovhNotificationsSidebar from '@ovh-ux/manager-notifications-sidebar';
```

Comment:
**thought**: The package name could be `@ovh-ux/manager-notifications` and expose different component to handle notifications. We currently use a sidebar but this could evolve

Reply:
this module is temporary (i created it so i can rewrite the navbar as a standalone react module without having to fully rewrite notifications + account panel) but i think we'll drop it as soon as we create a fragment for displaying notifications. that's why i suggest that we don't invest much time into this one since it'll be deleted soon (hopefully!)